samples/wifi/ble_coex/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Disable Oberon PSA crypto drivers
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=n
+
+# Enable PSA crypto from SSF client
+CONFIG_PSA_SSF_CRYPTO_CLIENT=y
+CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED=y
+
+# Disable Data Cache
```

Comment:
Why?

Reply:
This is copy-pasted from crypto samples board files